backend/src/Services/GlobalData.ts

```diff
@@ -0,0 +1,9 @@
+export default class GlobalData {
+
+  serviceUrl:string;
+
+  constructor() {
+    this.serviceUrl = "https://star-vote.herokuapp.com/";
```

Comment:
Would it make sense for this to be an environment variable?

Reply:
Another great call :)